### PR TITLE
add Cloud Shell instructions

### PR DESCRIPTION
**Pull request by ebellm**

### CODE REVIEW

File: doc/playbook.rst [hunk 1]
```diff
@@ -127,6 +127,15 @@ It accepts messages and gateways them to Cloud Run.
 tester
 ======
 
+Run the tester either on a local machine, or in Cloud Shell.
```
Comment (kfindeisen) [inline doc/playbook.rst:130]: I think the text defining what the tester _is_ really should go first; it's not a very intuitive name. I suggest moving the new text between "It takes a JSON token..." and "Command line arguments are...".

File: doc/playbook.rst [hunk 2]
```diff
@@ -127,6 +127,15 @@ It accepts messages and gateways them to Cloud Run.
 tester
 ======
 
+Run the tester either on a local machine, or in Cloud Shell.  
+For the latter, install the protype code and the GCP pubsub client:
```
Comment (kfindeisen) [inline doc/playbook.rst:131]: Did you mean "for the former"? It seems strange that you'd have to do these steps (especially installing `google-cloud-pubsub`) on Google's own system, whereas they make sense on a local machine.

**Review by kfindeisen — COMMENTED** [review]
Thanks for adding this, but can you please clarify whether the example you give is for a local machine or Cloud Shell?

File: doc/playbook.rst (hunk 2 above)
Comment (kfindeisen) [reply doc/playbook.rst:131]: Never mind, these instructions are indeed for the Shell.

**Review by kfindeisen — APPROVED** [review]

**Comment by kfindeisen** [comment]
Can you also add some information on how to acquire the JSON token in the Cloud Shell?

File: doc/playbook.rst [hunk 3]
```diff
@@ -127,6 +127,15 @@ It accepts messages and gateways them to Cloud Run.
...
+
+.. code-block:: sh
+
+    gcloud config set project prompt-proto
+    git clone https://github.com/lsst-dm/prompt_prototype.git
+    pip3 install google-cloud-pubsub
+
 ``tester/upload.py`` is a script that simulates the CCS image writer.
 It takes a JSON token in ``./prompt-proto-upload.json``.
```
Comment (ktlim) [inline doc/playbook.rst:140]: We should modify it to not force the use of a JSON token when it's not necessary (such as in the Cloud Shell environment).

File: doc/playbook.rst (hunk 2 above)
Comment (ktlim) [inline doc/playbook.rst:131]:
```suggestion
For the latter, install the prototype code and the GCP pubsub client:
```

**Pull request merged**